src/themes/default/components/core/ProductTile.vue

```diff
@@ -57,6 +57,20 @@
         v-if="!product.special_price && parseFloat(product.price_incl_tax) > 0 && !onlyImage"
       >{{ product.price_incl_tax | price }}</span>
     </router-link>
+    <div v-if="canAddToCart">
```

Comment:
You are using prop to distinguish whether display or not add to cart button. So rename `addToCartFromListing` to `showAddToCart` and use that prop directly here.

Reply:
Fixed